gate-core/src/main/groovy/com/netflix/spinnaker/gate/services/commands/HystrixFactory.groovy

```diff
@@ -30,7 +30,7 @@ class HystrixFactory {
     HystrixCommandGroupKey.Factory.asKey(name)
   }
 
-  public static <T> ListCommand newListCommand(String groupKey,
+  public static <T> ListCommand<T> newListCommand(String groupKey,
```

Comment:
Lines below are unaligned now

Reply:
This whole file has strange pre-existing alignment; I can fix this function definition as I'm touching it.